src/go/k8s/tests/e2e-unstable/update-conf-image/03-assert.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       initContainers:
         - image: "vectorized/configurator:latest"
 status:
-  readyReplicas: 2
+  replicas: 2
```

Comment:
Are you sure that this shouldn't be readyReplicas?

Reply:
The problem is that the configurator used in the test has a fixed image and, over time it may become incompatible. So, by checking only `replicas`, we ensure just that the image is exchanged, while the check on `readyReplicas` will also test that it starts correctly. We can leave `readyReplicas`, but we need to keep the test updated as new versions are released

Reply:
In my view we should maintain this that as we release new versions. The readyReplicas will assures that Redpanda is running and the init configurator finished successfully.